doc/examples/transform/plot_histogram_matching.py

```diff
@@ -0,0 +1,46 @@
+"""
+==================
+Histogram matching
+==================
+
+This example demonstrates the feature of histogram matching. It manipulates the
+pixels of an input image so that its histogram matches the histogram of the
+reference image. If the images have multiple channels, the matching is done
+independently for each channel, as long as the number of channels is equal in
+the input image and the reference.
+
+Histogram matching can be used as a lightweight normalisation for image
+processing, such as feature matching, especially in circumstances where the
+images have been taken from different sources or in different conditions (i.e.
+lighting).
+"""
+
+import matplotlib.pyplot as plt
+
+from skimage import data
+from skimage.transform import match_histograms
+
+reference = data.astronaut()
+image = data.chelsea()
```

Comment:
I think we can improve the example by adding more suitable images, but I would suggest to do that in a separate PR.